aws-example/README.md

```diff
@@ -142,6 +142,7 @@ Statement:
 
    Downloaded files will collect in the "objects" folder.
    Files will be in [NetCDF](http://www.unidata.ucar.edu/software/netcdf/docs/netcdf_introduction.html) format and have diagnostics
-   that correspond to the arguments supplied in step 5.
+   that correspond to the arguments supplied in step 5. Note, however, that the filenames will be entirely random, i.e. details of
```

Comment:
:+1: for the wording,

It will also need to be added to the readme on the http example or on the top level readme

Reply:
OK, ta. Seems a bit specific for top-level, but have added to HTTP example